container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1883,34 +1871,22 @@ configuration:
     description: "log level: debug, info, error, fatal"
   - name: AUTOSCALER_INSTANCE_METRICS_DB_CONNECTION_CONFIG_CONNECTION_MAX_LIFETIME
     default: "60s"
-    description: "The maximum lifetime of instance metrics database connection in connection pool"
```

Comment:
Why can't we use one setting to control both `AUTOSCALER_APPMETRICS_DB_CONNECTION_CONFIG_CONNECTION_MAX_LIFETIME` and `AUTOSCALER_INSTANCE_METRICS_DB_CONNECTION_CONFIG_CONNECTION_MAX_LIFETIME`?

e.g.
`AUTOSCALER_DATABASE_CONNECTION_MAX_LIFETIME`